pkg/asset/cluster/tfvars.go

```diff
@@ -342,6 +342,7 @@ func (t *TerraformVariables) Generate(parents asset.Parents) error {
 				Auth:                            auth,
 				CloudName:                       installConfig.Config.Azure.CloudName,
 				ARMEndpoint:                     installConfig.Config.Azure.ARMEndpoint,
+				ClusterOSImage:                  installConfig.Config.Azure.ClusterOSImage,
```

Comment:
Ah, similar to as staebler pointed out in the other comment, I don't think this is used either.